integration-tests/src/test/java/com/redhat/parodos/integration/notification/NotificationRecordTest.java

```diff
@@ -44,12 +43,17 @@ public class NotificationRecordTest {
 	private int countNotificationsExpectedCount = 0;
 
 	@Before
-	public void setUp() throws IOException {
-		ApiClient apiClient = Configuration.getDefaultApiClient();
-		apiClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, "Basic " + CredUtils.getBase64Creds(user, password));
-		apiClient.setBasePath(BASE_PATH);
-		recordApiInstance = new NotificationRecordApi(apiClient);
-		messageApiInstance = new NotificationMessageApi(apiClient);
+	public void setUp() throws ApiException, InterruptedException {
+
+		NotificationTestBuilder.TestComponents components = new NotificationTestBuilder().build();
+		log.info("CHECK NotificationWorkFlowTask basePath: {}", components.apiClient().getBasePath());
+		OkHttpClient httpClient = components.apiClient().getHttpClient();
+		HttpLoggingInterceptor loggingInterceptor = new HttpLoggingInterceptor();
+		loggingInterceptor.setLevel(HttpLoggingInterceptor.Level.HEADERS);
+		httpClient.newBuilder().addInterceptor(loggingInterceptor);
+		components.apiClient().setHttpClient(httpClient);
```

Comment:
Can this be moved to `NotificationTestBuilder` as optional function before calling to `build()`
Same question to setting authentication below

Reply:
this was a leftover debugging code. I'll remove it.